Cargo.toml

```diff
@@ -32,7 +32,13 @@ optional = true
 version = ">=0.0"
 
 [dependencies.libc]
-version = "=0.1.12"
+version = "=0.2.11"
```

Comment:
Can you make this, and the dependencies below `0.2.*` etc, rather than pinning them to the specific version please?